src/Engine/ProtoImperative/CodeGen.cs

```diff
@@ -2817,24 +2817,6 @@ private void EmitRangeExprNode(ImperativeNode node, ref ProtoCore.Type inferedTy
                 }
             }
 
-            // Replace with build-in RangeExpression() function. - Yu Ke
-            var tmpFrom = nodeBuilder.BuildTempVariable();
```

Comment:
Not sure what the original intention was, but i dont think it was for SSA. If we want to SSA in imperative, this should have been handled by BuildSSA(...). It should be ok to remove